src/blocks/pagination/Pagination.types.ts

```diff
@@ -0,0 +1,12 @@
+export type PaginationProps = {
+  //onChange function
```

Comment:
Use this format with proper explanation.

/* This is a proper comment with proper explanation. */

Reply:
for prop naming please refer to: https://ant.design/components/pagination